test/tools/TestMetadata.json

```diff
@@ -1,7 +1,5 @@
 {
   "ExperimentalFeatures": {
-    "ExpTest.FeatureOne": [ "test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1" ],
```

Comment:
Ideally, you should add entries for those features that are still kept as experimental, because otherwise, their tests will not be evaluated during the GA release build.
But their tests will still be run in our daily build, so that can be deferred to another PR.